recipes/sagemaker/meta.yaml

```diff
@@ -0,0 +1,71 @@
+{% set name = "sagemaker-python-sdk" %}
+{% set version = "1.43.4" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://github.com/aws/{{ name }}/archive/v{{ version }}.tar.gz"
+  sha256: 72ae19ac61932bb91319af100f5750016eadea414d9091d44d27d49b934e5fd7
+  patches:
+    - remove_docker_compose_fromsetup.patch # build will not work unless docker-compose is removed from setup.py, it will produce erroneous ContextualVersionConflicts
+
+build:
+  number: 0
+  entry_points:
+    - sagemaker=sagemaker.cli.main:main
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [win or py35 or py37]
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - boto3 >=1.9.213
+    - docker-compose >=1.23.0
+    - fabric >=2.0
+    - numpy >=1.9.0
+    - protobuf >=3.1
+    - protobuf3-to-dict >=0.1.5
+    - python
+    - requests >=2.20.0,<2.21
+    - scipy >=0.19.0
+    - urllib3 >=1.21,<1.25
+    - enum34 >=1.1.6  # [py<34]
+
+test:
+  imports:
+    - sagemaker
+    - sagemaker.amazon
+    - sagemaker.chainer
+    - sagemaker.cli
+    - sagemaker.local
+    - sagemaker.mxnet
+    - sagemaker.pytorch
+    - sagemaker.rl
+    - sagemaker.sklearn
+    - sagemaker.sparkml
+    - sagemaker.tensorflow
+    - sagemaker.tensorflow.tensorflow_serving
+    - sagemaker.tensorflow.tensorflow_serving.apis
+    - sagemaker.workflow
+    - sagemaker.xgboost
+  commands:
+    - sagemaker --help
+
+about:
+  home: "https://github.com/aws/sagemaker-python-sdk/"
+  license: Apache Software
+  license_family: APACHE
+  license_file: LICENSE.txt
```

Comment:
Please find the exact identifier here: https://spdx.org/licenses/

Reply:
thanks! i've just updated to `Apache-2.0`